rllib/core/rl_module/marl_module.py

```diff
@@ -15,6 +15,21 @@
 ModuleID = str
 
 
+def _get_module_configs(config: Dict[str, Any]):
+    """Constructs a mapping from module_id to module config.
+
+    It takes care of the inheritance of common configs to individual module configs.
+    See `from_multi_agent_config` for more details.
```

Comment:
Args? Returns?

Reply:
The documentation will be a duplication of what's already in `from_multi_agent_config` documentation. So I tried to be concise and not redundant here.